src/ensembl/src/genome/genomeActions.ts

```diff
@@ -1,17 +1,24 @@
 import { createAsyncAction } from 'typesafe-actions';
 import { ThunkAction } from 'redux-thunk';
 import { Action, ActionCreator } from 'redux';
-
+import find from 'lodash/find';
 import apiService from 'src/services/api-service';
 import { RootState } from 'src/store';
-import { GenomeInfoData, GenomeTrackCategories } from './genomeTypes';
+import {
+  GenomeInfoData,
+  GenomeInfo,
+  GenomeTrackCategories
+} from './genomeTypes';
+import omit from 'lodash/omit';
```

Comment:
according to lodash's roadmap for v5 (which will hopefully happen some time), they are going to remove `omit` and just use pickBy: https://github.com/lodash/lodash/wiki/Roadmap#v500-2019

Can we avoid  lodash's `omit` as well? (I personally love it, but so that we don't have to replace it later?)

Reply:
Updated it. Thanks 👍